webpack/subscriptions_extension_index.js

```diff
@@ -0,0 +1,8 @@
+import { subscriptionCountListener } from './ForemanInventoryUpload/SubscriptionsPageExtension/SubscriptionsPageExtensionActions';
+
+if (window.location.pathname === '/subscriptions') {
+  window.tfm.store.observeStore(
```

Comment:
since this is being called each time the subscription page loads,
I think it worth to call the component here as  `subscriptionCountListener()`
and connect the component to redux.

can we also check here if the `auto_upload` setting is turned `off`? otherwise we don't need to load the function/component